llarp/config/config.cpp

```diff
@@ -960,13 +960,31 @@ namespace llarp
   void
   Config::Save()
   {
+    if (not fs::exists(m_overridesDir))
+      fs::create_directory(m_overridesDir);
     m_Parser.Save();
   }
 
   void
   Config::Override(std::string section, std::string key, std::string value)
   {
-    m_Parser.AddOverride(std::move(section), std::move(key), std::move(value));
+    m_Parser.AddOverride(m_overridesDir / "overrides.ini", section, key, value);
+  }
+
+  void
+  Config::LoadOverrides(fs::path defaultDataDir)
+  {
+    m_overridesDir = defaultDataDir / "conf.d";
```

Comment:
If `m_overridesDir` is only being used in this function, it shouldn't be a member variable.  If it should be a member variable, it should maybe be set elsewhere?